src/components/MaterialUI/DataDisplay/PredefinedElements/Translation.js

```diff
@@ -0,0 +1,78 @@
+import React, { useMemo } from "react";
+import { FormattedMessage } from "react-intl";
+import { makeStyles } from "@material-ui/core/styles";
+import CollapsibleList from "../CollapsableList";
+import CollapsibleListProps from "../collapsableListProps";
+import { InputBase, InputBaseProps } from "../../Inputs";
+import sharedMessages from "../../../../sharedMessages";
+
+export const useStyles = makeStyles(theme => ({
+	elementContainer: {
+		marginBottom: theme.spacing(1.5),
+	},
+}));
+
+const Translation = ({ cultures, defaultCulture, value = {}, update, disabled = false, errors = undefined }) => {
+	const classes = useStyles();
+	const collapsibleListProps = new CollapsibleListProps();
+
+	collapsibleListProps.set(CollapsibleListProps.propNames.hasMessage, true);
+	collapsibleListProps.set(
+		CollapsibleListProps.propNames.openMessage,
+		<FormattedMessage {...sharedMessages.showMoreLanguages} />,
+	);
+	collapsibleListProps.set(
+		CollapsibleListProps.propNames.closeMessage,
+		<FormattedMessage {...sharedMessages.showFewerLanguages} />,
+	);
+	collapsibleListProps.set(CollapsibleListProps.propNames.expandPosition, "right");
+	collapsibleListProps.set(CollapsibleListProps.propNames.containerWidth, "unset");
+
+	const handlers = useMemo(
+		() =>
+			cultures.reduce((handlers, lang) => {
+				handlers[lang] = fieldValue => update({ ...value, [lang]: fieldValue });
+				return handlers;
+			}, {}),
+		[update, cultures, value],
+	);
+
+	const elementList = cultures
+		.sort((ca, cb) => {
+			if (ca === defaultCulture) return -1;
+			if (cb === defaultCulture) return 1;
+			return 0;
+		})
+		.map(lang => {
+			const inputProps = new InputBaseProps();
+			inputProps.set(InputBaseProps.propNames.label, lang);
+			inputProps.set(InputBaseProps.propNames.value, value[lang]);
+			inputProps.set(InputBaseProps.propNames.update, handlers[lang]);
+			inputProps.set(InputBaseProps.propNames.disabled, disabled);
+			if (defaultCulture && defaultCulture === lang && errors) {
```

Comment:
Use next logic:
If errors[lang] is present , then we assign errors for corresponding  InputBase. 
inputProps.set(InputBaseProps.propNames.error, errors[lang]);